tests/conftest.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from django.contrib.auth.models import AnonymousUser
+from django.contrib.sessions.middleware import SessionMiddleware
+from django.test import RequestFactory
+
+from factories import (UserFactory, InstitutionFactory)
+
+
+@pytest.fixture
+def user(uai_number):
+    user = UserFactory(
+        id=1,
+        email='noel@flantier.com',
```

Comment:
This should go in the factory, with a sequence to increment the email (in order to have some unique emails).

Like this
```python
email = factory.Sequence(lambda n: 'person{0}@example.com'.format(n))
```

Reply:
Fixed by a37b1fd